src/modules/Channel/components/Message/index.tsx

```diff
@@ -49,9 +49,14 @@ const Message = (props: MessageProps): React.ReactElement => {
         !initialized || isDisabledBecauseFrozen(currentGroupChannel) || isDisabledBecauseMuted(currentGroupChannel) || !config.isOnline
       }
       shouldRenderSuggestedReplies={
-        message.messageId === currentGroupChannel?.lastMessage?.messageId
+        config?.groupChannel?.enableSuggestedReplies
+        && message.messageId === currentGroupChannel?.lastMessage?.messageId
         // the options should appear only when there's no failed or pending messages
-        && localMessages.every((message) => (message as UserMessage).sendingStatus === 'succeeded')
+        && (
+          !localMessages
+          || localMessages.length === 0
+          || localMessages.every((message) => (message as UserMessage).sendingStatus === 'succeeded')
+        )
```

Comment:
In my opinion, using `localMessages?.length === 0` seems sufficient. What do you think?